config/game_options.txt

```diff
@@ -397,15 +397,15 @@ NEAR_DEATH_EXPERIENCE
 ## Set to 1 for "custom", silicons will start with the custom laws defined in silicon_laws.txt. (If silicon_laws.txt is empty, the AI will spawn with asimov and Custom boards will auto-delete.)
 ## Set to 2 for "random", silicons will start with a random lawset picked from random laws specified below.
 ## Set to 3 for "weighted random", using values in "silicon_weights.txt", a law will be selected, with weights specifed in that file.
-DEFAULT_LAWS 2
+DEFAULT_LAWS 3
 
 ## RANDOM LAWS ##
 ## ------------------------------------------------------------------------------------------
 ## These control what laws are available for selection if random silicon laws are active.
 ## See datums\ai_laws.dm for the full law lists
 
 ## standard-ish laws. These are fairly ok to run
-RANDOM_LAWS asimov
+RANDOM_LAWS crewsimov
```

Comment:
Doubled up crewsimov.

Reply:
My bad, someone mentioned in admin chat that we shouldn't have stock asimov as a normal lawset with the number of non-humans we have.  Fixed to simply remove asimov from the roll